src/torch_tomogram/tomogram.py

```diff
@@ -27,13 +34,95 @@ def __init__(
         self.sample_translations = torch.as_tensor(
             sample_translations, device=device
         ).float()
+        self.pixel_spacing = pixel_spacing
         self.device = device
         self._pad_factor = 2.0
+    
+    @property
+    def sample_translations_px(self) -> torch.Tensor:
+        """Get sample translations in pixels.
+        
+        If pixel_spacing was provided, converts from Angstroms to pixels.
+        Otherwise, returns the translations as-is (already in pixels).
+        """
+        if self.pixel_spacing is not None:
+            return self.sample_translations / self.pixel_spacing
+        return self.sample_translations
+
+    @classmethod
+    def from_aretomo_aln(
+        cls,
+        aln_path: Path | str,
+        tilt_stack_path: Path | str,
+        pixel_spacing: float | None = None,
+        device: torch.device | str = "cpu",
+    ) -> "Tomogram":
+        """Initialize Tomogram from AreTomo alignment file and tilt stack."""
+
+        aln_path = Path(aln_path)
+        tilt_stack_path = Path(tilt_stack_path)
+        df = alnfile.read(aln_path)
+        corrected_shifts_xy = df[["tx", "ty"]].to_numpy()
+        corrected_shifts_yx = corrected_shifts_xy[:, ::-1].copy()
+        
+        # Convert shifts to Angstroms if pixel_spacing provided
+        if pixel_spacing is not None:
+            corrected_shifts_yx = corrected_shifts_yx * pixel_spacing
+        
+        tilt_stack_full = mrcfile.read(tilt_stack_path)
+        included_indices = df["sec"].values - 1 
```

Comment:
I don't understand why this makes sense, maybe a comment would be useful

alnfile docs say `sec` corresponds to zero indexed indices into the tilt stack, this ` - 1` implies the values are one indexed?

also should be made clear that stack is expected to be aretomo output stack (avoid users supplying aretomo input stack and this code implicitly accounting for dark frames which haven't been removed from that stack)

```suggestion
        idx_valid = df["sec"].values - 1 
```

Reply:
Apologies, my mistake. I incorrectly wrote '0 indexed' in the alnfile readme, but it is actually '1 indexed'.

Reply:
no big deal, glad we caught it in review! can hopefully avoid someone getting confused in the future 🙂

Reply:
Actually, it looks like AreTomo3 switched the SEC indexing from 0-based to 1-based starting in v2.0.10. This isn’t documented in their manual yet, but I noticed this inconsistency by testing alnfile on different versions (I think this is the commit when they introduced ToOneBased: https://github.com/czimaginginstitute/AreTomo3/commit/048b161)

I’ll update alnfile so it automatically handles both conventions internally, while keeping the SEC output normalized to 0-based. For most downstream applications, 0-based indexing seems to be the more practical and consistent choice, but if you have a preference for keeping it 1-based for some workflows, I’m happy to discuss it!